packages/tools/client-debugger/client-debugger-chrome-extension/src/devtools/BackgroundConnection.ts

```diff
@@ -60,14 +61,20 @@ export class BackgroundConnection
 	private readonly backgroundServiceConnection: TypedPortConnection;
 
 	public static async Initialize(): Promise<BackgroundConnection> {
-		const connection = new BackgroundConnection();
+		const connection = new BackgroundConnection(extensionMessageSource);
 		await new Promise((resolve) => {
 			connection.once("tabConnected", resolve);
 		});
 		return connection;
 	}
 
-	private constructor() {
+	/**
+	 * Creates an instance of {@link BackgroundConnection}.
+	 *
+	 * @param messageSource - All messages sent through the returned instance's {@link BackgroundConnection.postMessage}
```

Comment:
Nit: I think if you make this a comment on the inline property, it will bubble up automatically as a param comment for the constructor.